R/getConstraints.R

```diff
@@ -41,9 +41,6 @@
 #' @export
 getConstraints = function(formula, data, props = NULL){
 
-	# Fill in the data argument with stats::model.frame
```

Comment:
Is it not possible to include this here? Since getConstraints() is a standalone function, it would be great if 'data' were an optional argument.